queries/landuse.jinja2

```diff
@@ -86,6 +86,7 @@ SELECT
     osm_id AS __id__,
     sport,
     religion,
+    surface,
```

Comment:
I think you'll need to include this column in the union query for zoom level 9 too.